deno.json

```
@@ -0,0 +1,29 @@
+{
+  "tasks": {
+    "npm:install": "deno run -A npm:npm install",
+    "lint": "deno run -A npm:eslint src --ext .js,.jsx,.ts,.tsx,.astro"
+  },
+  "lint": {
+    "include": ["src/"],
+    "exclude": ["node_modules/", "dist/", ".astro/"],
+    "rules": {
+      "tags": ["recommended"],
```

```suggestion
      "tags": ["recommended", "jsx"],
```
Agregar reglas recomendables a JSX